upload-api/test/helpers/ucan.js

```diff
@@ -295,6 +295,7 @@ export async function executionContextToUcantoTestServerContext (t) {
 
   return {
     ...serviceContext,
+    grantAccess: async () => {},
```

Comment:
```suggestion
```

Reply:
oh lol we were looking at this at precisely the same time - fixed, thx!